client/mobile/components/teacher/display_feedback/feedbackView.js

```diff
@@ -121,16 +123,14 @@ class FeedbackView extends React.Component {
     //need back/cancle button
     return (
       <View style={{flex: 1, backgroundColor: '#ededed'}}> 
+      <View>
+        <NavBar navi={this.props.navigator} onBack={this.previousSection.bind(this)}>{this.state.feedbackOption.name}</NavBar>
+      </View>
         <View style={styles.viewContainer}>
+        {this.renderChart.call(this)}
         <View style={styles.backButtonContainer}>
-          <TouchableOpacity onPress={this.exitPage.bind(this)} style={styles.backButton}>
-            <Text style={styles.backButtonText}> Save and Exit </Text>
-          </TouchableOpacity>
+          <Button onPress={this.previousSection .bind(this)} style={styles.backButton} text={'Save and Exit'} />
```

Comment:
We have a 'save and exit' and a back on the navigator. What is the difference in functionality? Should we change the names of these buttons at all?